docs/content/tutorial.md

```diff
@@ -0,0 +1,142 @@
+      - This is another list item
+
+      ---
+
+      # Second Slide
+
+      Text can be **bold** or _italic_!
+
+      Notes: These are presenter notes, only visible in presenter mode to the speaker.
+      ```
+
+    **Note:** The triple dash (`---`) is used as a slide delimiter. The `Notes:` keyword is used to embed presenter notes only visible to the speaker in presenter mode.
+
+2. To view your slides, supply your markdown to the Spectacle CLI to start a local web server.
+
+    ```sh
+    $ npm install --global spectacle-cli
+    $ spectacle -s my-slides.mdx
+    ```
+
+3. And you're good to go! The web server you started supports live refreshing and will update your deck as you make changes to the markdown file.
+
+<a name="option-three"></a>
+
+## Option Three: Using One Page
+
+One Page is a single self-contained `HTML` file that lets you build a deck using no build steps, using [htm](https://github.com/developit/htm) over JSX to reduce the dependencies and load time.
+
+As a self-contained entity, it already has references to the dependencies you need to author and launch a deck in a web browser. Since there is no tooling required, One Page is also optimal on tablets. The One Page `HTML` file can be downloaded from the `examples` directory [in this repository](https://unpkg.com/browse/spectacle@latest/examples/one-page.html).
+
+<a name="next-steps"></a>
+
+## Next Steps
+
+<a name="styling"></a>
+
+### Styling your Spectacle Deck
+
+The easiest way to apply consistent styles to your Spectacle deck is using [themes](./advanced-concepts#themes).
+
+1. Create a theme JS file containing a single object export. Supplied properties will be merged with the default base theme (found in Spectacle at `src/theme/default-theme.js`). 
+
+    Here's a sample object:
+
+    ```js
+    export default {
+      colors: {
+        primary: 'red',
+        secondary: 'green'
+      },
+      fonts: {
+        header: '"Helvetica Neue", Helvetica, Arial, sans-serif'
+      },
+      fontSizes: {
+        h1: '72px',
+        h2: '64px'
+      }
+    };
+    ```
+
+2. Consume the theme using the approach of your choice:
+
+    a. To use a custom theme with a JSX (Option One) or HTM-deck (Option Three), supply the object to the `theme` prop in the `Deck` tag. `<Deck theme={customTheme}>`.
+
+    b. To use a custom theme with the Markdown CLI (Option Two), supply the file using the `-t` argument.
+
+      ```sh
+      $ npm install --global spectacle-cli
+      $ spectacle -s my-slides.mdx -t custom-theme.js
+      ```
+
+<a name="sharing"></a>
+
+### Sharing your Spectacle Deck
+
+For more information on [presenting](./basic-concepts#presenting), [exporting](./advanced-concepts#exporting), [building](./advanced-concepts#build--deployment), or [deploying](./advanced-concepts#build--deployment) your Spectacle deck, please check out [the documentation on advanced concepts](https://formidable.com/open-source/spectacle/docs/advanced-concepts/).
```

Comment:
All of these links break for me with the `TypeError: Cannot read property 'scrollIntoView' of undefined` thing which I think is just a 404 because the link path ends up incorrect... (Like resolved link ends up `http://127.0.0.1:3000/docs/tutorial/basic-concepts#presenting` instead of `http://127.0.0.1:3000/docs/basic-concepts#presenting`)

Reply:
That link seems odd 🤔 Shouldn't `/docs/advanced-concepts` be a valid route on its own? Why does the page need to be tacked on to the `docs/tutorial` route?

Reply:
We shouldn’t do FQDN. There’s like a one level directory issue I think instead when run through react static. You most likely need an extra ..

Reply:
We shouldn’t do FQDN. There’s like a one level directory issue I think instead when run through react static. You most likely need an extra ..

Reply:
We shouldn’t do FQDN. There’s like a one level directory issue I think instead when run through react static. You most likely need an extra ..

Reply:
This works:

```diff
diff --git a/docs/content/tutorial.md b/docs/content/tutorial.md
index cd501a1..28c21f3 100644
--- a/docs/content/tutorial.md
+++ b/docs/content/tutorial.md
@@ -129,7 +129,7 @@ The easiest way to apply consistent styles to your Spectacle deck is using [them
 
 ### Sharing your Spectacle Deck
 
-For more information on [presenting](./basic-concepts#presenting), [exporting](./advanced-concepts#exporting), [building](./advanced-concepts#build--deployment), or [deploying](./advanced-concepts#build--deployment) your Spectacle deck, please check out [the documentation on advanced concepts](https://formidable.com/open-source/spectacle/docs/advanced-concepts/).
+For more information on [presenting](./basic-concepts#presenting), [exporting](./advanced-concepts#exporting), [building](./advanced-concepts#build--deployment), or [deploying](./advanced-concepts#build--deployment) your Spectacle deck, please check out [the documentation on advanced concepts](../advanced-concepts/).
 
 <a name="documentation-contributing-and-source"></a>
 
```

The change was to `../advanced-concepts/`